app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
The function `go_to_cafe` is missing type annotation for its return value. According to the checklist, it's a good practice to use type annotations. Here is how you can do it: `def go_to_cafe(friends: list, cafe: Cafe) -> str:`